scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala

```diff
@@ -1,31 +1,27 @@
 package org.scalafmt.util
 
-import scala.collection.mutable
-import scala.meta.Tree
-import scala.meta.parsers.Parse
-import scala.meta.parsers.ParseException
-import scala.util.Try
-
 import java.io.File
 
 import metaconfig.Configured
-import org.scalafmt.Debug
+import munit.Assertions._
 import org.scalafmt.Error.UnknownStyle
-import org.scalafmt.Scalafmt
-import org.scalafmt.config.Config
-import org.scalafmt.config.ContinuationIndent
-import org.scalafmt.config.DanglingParentheses
-import org.scalafmt.config.FormatEvent.CompleteFormat
-import org.scalafmt.config.FormatEvent.CreateFormatOps
-import org.scalafmt.config.FormatEvent.Enqueue
-import org.scalafmt.config.FormatEvent.Explored
-import org.scalafmt.config.FormatEvent.VisitToken
-import org.scalafmt.config.ScalafmtConfig
-import org.scalafmt.config.ScalafmtRunner
+import org.scalafmt.{Debug, Scalafmt}
+import org.scalafmt.config.FormatEvent._
+import org.scalafmt.config.{
+  Config,
+  ContinuationIndent,
+  DanglingParentheses,
+  ScalafmtConfig,
+  ScalafmtRunner
+}
 import org.scalafmt.internal.FormatWriter
-import org.scalatest.funsuite.AnyFunSuiteLike
 
-trait HasTests extends AnyFunSuiteLike with FormatAssertions {
+import scala.collection.mutable
+import scala.meta.Tree
+import scala.meta.parsers.Parse
+import scala.util.Try
+
+trait HasTests extends FormatAssertions {
```

Comment:
You can btw mix FunSuite by adding a self parameter “{this: FunSuite => ...”

Reply:
This means you can only implement this trait from a subclass or FunSuite

Reply:
Reasonable constraint

Reply:
No, we can do it now, because of `ManualTests` and `UnitTests` [objects](https://github.com/poslegm/scalafmt/blob/a90ca60f778c48fd1fd13c312500c5a4b2f16a3c/scalafmt-tests/src/test/scala/org/scalafmt/ManualTests.scala#L8) which extends this trait.

So, we need to small redesign this part of code before introducing `this: FunSuite =>` here. I think it's not necessarily now 😄